### ISSUE TITLE
Excel多Sheet 导入模板生成出错

### ISSUE THREAD

**Issue by AjuPrince** [dotnetcore/Magicodes.IE#133]
Excel多Sheet 导入模板生成出错，好像不支持，还是如何使用？

**Comment by hueifeng** [comment]
Multiple Sheet exports can be referenced https://github.com/dotnetcore/Magicodes.IE/blob/master/docs/6.%E5%A4%9ASheet%E5%AF%BC%E5%85%A5.md , If there is a problem, you can feedback the details to us.

**Comment by AjuPrince** [comment]
![image](https://user-images.githubusercontent.com/16829343/89381547-ee4fe380-d72b-11ea-93fb-4c1c1341f986.png)
这两个Sheet是如何生成的？
我使用
![image](https://user-images.githubusercontent.com/16829343/89381671-1b9c9180-d72c-11ea-95ad-470a1c0b86bc.png)这个方法生成模板后，是不对的？
Dto类如下：
![image](https://user-images.githubusercontent.com/16829343/89381774-3ec74100-d72c-11ea-93a8-c56a7c4d7c10.png)

**Comment by hueifeng** [comment]
SheetName needs to be changed.

```
ExcelImporter(SheetName="1班导入数据")
```

**Comment by AjuPrince** [comment]
![image](https://user-images.githubusercontent.com/16829343/89382244-1e4bb680-d72d-11ea-8756-d5329a374435.png)

如上图所示，是更改了名称的

**Comment by hueifeng** [comment]
Generating multiple Sheet templates is not supported, currently only multiple Sheet imports and exports are supported.

**Comment by hueifeng** [comment]
We released 2.3.0-beta6 which can be updated in the following way.
```
PM> Install-Package Magicodes.IE.Excel -Version 2.3.0-beta6
```
